### DISCUSSION TITLE
picture printing problem

### DISCUSSION THREAD

**Discussion by Oleksij79** [arkypita/LaserGRBL#2362] [Polls]
![photo_2024-09-17_13-31-26](https://github.com/user-attachments/assets/d974a248-8a00-4f38-a614-0f9e41bb6ed7)
for some reason, when printing a picture, it always leaves such a line

**Comment by Tribo-2dot0** [comment]
Verifique a integridade dos dentes da correia dentada e também se não há nada nos trilhos que possa obstruir.

**Reply by Oleksij79** [reply]
Thank you
![0-02-05-295be68a7ffee656d1f308ffb4f0c382712cc85b7755b5432cc30af9fd81bbb4_991191d31f928fa1](https://github.com/user-attachments/assets/fb5c66c9-0717-4fe6-96a1-eaf78f0ba001)
 the wires came off when the typewriter moved away to such a distance that the cord was stretched, then the microcircuit tilted a little, and if the typewriter went astray, I was better at fixing them
